CHANGELOG.md

```diff
@@ -4,7 +4,15 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.0.0] - Unreleased
+## [1.1.1] - Unreleased
```

Comment:
fyi - As 2.0.0 is never released, based on the convo we had on NOT considering the relaxation of rails constraint as breaking change, I chose to jump bump the minor version. Thoughts?

Reply:
Minor version bump sounds right to me. 👍

Reply:
I agree with a minor version as well. Since `1.1.0` wasn't released, can we combine these changes with the `1.1.0` release rather than making an arbitrary patch version bump?